components/ActivityIndicator/README.md

```diff
@@ -11,10 +11,12 @@ api_doc_root: true
 # Activity Indicators
 
 <div class="article__asset article__asset--screenshot">
-  <img src="docs/assets/activity_indicator.png" alt="Activity Indicator" width="375">
+  <img src="docs/assets/activityindicator.gif" alt="Activity Indicator" width="284">
```

Comment:
The new gif seems to have a lot of translucent pixels. Can we reduce the dimensions to reduce the file size?

Reply:
Done.